src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/CreateInductionTest.kt

```diff
@@ -135,195 +116,12 @@ class CreateInductionTest : IntegrationTestBase() {
 
   @Test
   @Transactional
```

Comment:
I'm guessing that once we've implemented the GET endpoint and written ITs for it, then we can do away with the `@Transactional` annotations and talking to the persistence layer directly, and do it via the REST API calls?

Reply:
Yes we can, although it would kind of negate the need for a `GetInductionTest`! 😄